iceoryx_posh/source/roudi/roudi.cpp

```diff
@@ -163,8 +163,8 @@ void RouDi::processRuntimeMessages() noexcept
 {
     runtime::IpcInterfaceCreator roudiIpcInterface{IPC_CHANNEL_ROUDI_NAME};
 
-    // the logger is intentionally not used, to ensure that this message is always printed
-    std::cout << "RouDi is ready for clients" << std::endl;
+    IOX_LOG(INFO) << "RouDi is ready for clients";
+    fflush(stdout); // explicitly flush 'stdout' for 'launch_testing'
```

Comment:
@dkroenke `fflush` could also be called in the logger itself but that would have quite an performance impact. Are you okay with calling `fflush` here or do you think it should be hidden behind a logger call, e.g. `iox::log::Logger::get().forceFlush();`?

cc @elfenpiff